projects/canopy/src/lib/forms/radio/radio.interface.ts

```diff
@@ -1 +1,2 @@
-export type RadioVariant = 'radio' | 'filter';
+export type RadioVariant = 'radio' | 'filter' | 'segment';
+export type RadioStack = 'sm' | 'md' | 'lg';
```

Comment:
This is great 👍   Just tested it out in Storybook. The only thing is that first I found the name a bit confusing at first, and had to get my head around the code before I understood exactly what it does. I wonder if we could rename this to `RadioStackBreakpoint`?

Also, I think this is only used for the Segment right? Should we consider including that in the name too? Maybe `RadioSegmentStackBreakpoint`. A bit long I suppose 🤔 

Also I think it would be a good idea to include a short paragraph in the notes about how the stacking works. 

Maybe at some point we should consider separate note files for each of the radio button styles, so we can be more description on their specific features and use-cases.

Reply:
Maybe we can just call it `RadioBreakpoint`? 
The idea is to expand this to all the radio and checkbox variants and get rid of the `inline` input.

Regarding the additional paragraph in the notes, have you seen that I've included the explanation in the `LgRadioGroupComponent` inputs? do you think it needs a bit more text?

Reply:
I'm inclined to agree with the renaming of this one, `RadioStackBreakpooint` was the first thing that jumped to my mind, it's a bit long but descriptive enough.

Reply:
I think in terms of the text we should add a bit more, maybe linked to this https://github.com/Legal-and-General/canopy/pull/86#issuecomment-738121194

Reply:
I quite like `RadioStackBreakpooint` too, it makes it very clear 👍 

In terms of the notes, maybe we could add a small paragraph in the "Purpose" section, something along the lines of..

```
Radios are displayed inline, unless given a `RadioStackBreakpoint`, which tells them at which breakpoint should they appear stacked on top of each other (currently only implemented for the Segment component).
```

What do you think?

Reply:
Cool, I'll rename the type to `RadioStackBreakpooint` 👍 

Regarding the notes, I haven't made the `stack` input visible to anything other than the segment's notes because we don't want people to know about it and use it incorrectly in the other variants. I will add a new section to the segments notes to explain that functionality a bit better as you suggested 👍

Reply:
Updated.